internal/configs/transportserver.go

```diff
@@ -305,6 +327,7 @@ func generateStreamUpstream(upstream conf_v1.TransportServerUpstream, upstreamNa
 		Name:                name,
 		Servers:             upsServers,
 		LoadBalancingMethod: generateLoadBalancingMethod(upstream.LoadBalancingMethod),
+		BackupServers:       upsBackups,
```

Comment:
These should be sorted to ensure the config generation is reproducable, e.g: 
```go
sort.Slice(upsBackups, func(i, j int) bool {
  return upsBackups[i].Address < upsBackups[j].Address
})

```

Reply:
done